index.html

```diff
@@ -24,7 +24,7 @@
         </div>
     </header>
```

Comment:
![color1](https://user-images.githubusercontent.com/93039359/187473797-16ddd57e-a9cc-4697-a3de-f977326da00f.png)
- [ ] On line 16 you have your logo which doesn't have a W3C level color contrast. Please make the color contrast upto 4.5:1 ratio :+1: